molgenis-js/src/main/resources/js/script-evaluator.js

```diff
@@ -6,22 +6,22 @@
  * @returns the evaluated script result
  */
 function evalScript (script, entity) {
-    // make functions available to the script in local scope
-    // noinspection JSUnusedLocalSymbols
-    var $ = MagmaScript.$.bind(entity) //NOSONAR
-    // noinspection JSUnusedLocalSymbols
-    var newValue = MagmaScript.newValue // NOSONAR
-    // noinspection JSUnusedLocalSymbols
-    var _isNull = MagmaScript._isNull // NOSONAR
-    return eval(script)
+  // make functions available to the script in local scope
```

Comment:
Was the previous formatting incorrect? Or is your formatting incorrect?

Reply:
The previous formatting was incorrect.